policyengine_us/variables/gov/states/ny/nyserda/drive_clean/drive_clean_vehicle_cost.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+class drive_clean_vehicle_cost(Variable):
+    value_type = float
+    entity = TaxUnit # TODO: Check if this is right. Not a tax credit.
+    label = "Vehicle Price"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.nyserda.ny.gov/All-Programs/Drive-Clean-Rebate-For-Electric-Cars-Program"
```

Comment:
If this is a definition that is given specifically by nyserda lets move to the nyserda folder and add `nyserda_` to the class and file names